app/views/pages/about.html.haml

```diff
@@ -11,22 +11,41 @@
-        %h3{class: 'govuk-heading-m govuk-!-padding-top-5'} Trusted by government services
-        %p By using registers you can:
-        %ul.list.list-bullet
+        %p A GOV.UK register is a structured dataset of government information.
+        %p Each register:
+        %ul.govuk-list.govuk-list--bullet
+          %li only contains data on a specific subject
+          %li is kept up-to-date by a subject matter expert from the relevant government organisation
+          %li shares a common API that supports JSON and CSV
+
+        %h3.govuk-heading-m Who registers are for
+        %p Registers are designed to be used by government and public sector services.
+        %p For example, the #{link_to 'country register', register_path('country')} lists all countries recognised by the UK government. It is updated by a subject matter expert, called a custodian, at the Foreign and Commonwealth Office. GOV.UK Pay uses the country register and #{link_to 'territory register', register_path('territory')} to populate an autocomplete feature that helps users enter their location.
+
+        %figure{class: 'govuk-!-margin-bottom-6 registers-aspect-3:2'}
+          = image_tag "country-picker.gif", class: 'aspect-inner'
+
+        %p Services using registers can:
+        %ul.govuk-list.govuk-list--bullet
           %li reduce the time and cost of sourcing data from across government
-          %li be confident that your service is using the most up&ndash;to&ndash;date government data
           %li receive data that is ready to use with no need for data cleansing
+          %li be confident that their service is using the most up-to-date government data
+
+        %h3.govuk-heading-m How to access a register
+        %p All registers can be accessed and used freely by anyone.
+        %p You can #{link_to 'use our API', 'https://docs.registers.service.gov.uk/#gov-uk-registers-technical-documentation'} to automatically pull data from a register into your service. This means you’ll always use the most recent version of the data in a register.
+        %p You can also download a copy of the latest data in each register.
+        %p All registers are available in JSON and CSV formats.
+        %p You can see all the registers that are ready to use in our #{link_to 'government registers collection', registers_path}.
+
+        %h3.govuk-heading-m Who is using registers
+        %p You can see which #{link_to 'government services', services_using_registers_path} are already using registers.
+        %p You can also find out which #{link_to 'government organisations', registers_path(show_by: "organisation")} have created a register with their data.
+
       %nav{class: 'govuk-grid-column-one-third'}
         = render partial: 'layouts/secondary-navigation'
-      %aside{class: 'govuk-grid-column-two-thirds govuk-!-padding-top-4'}
-        = render partial: 'layouts/registers-in-numbers'
```

Comment:
Is this partial still used somewhere else or can it be deleted?

Reply:
Good spot - thank you. It's now deleted.

Reply:
Seems like there are some other references e.g. https://github.com/openregister/registers-frontend/blob/5950aabbda8f87aacc220b2205b6a94d0e7fb5ff/app/controllers/pages_controller.rb#L29 please can you check if they can be safely deleted?

Reply:
The only place those variables were used was in the now-removed partial - so I've removed the variables as well.

Reply:
Um - The partial was only being used in one place 😳 

I've removed the file and placed the code in the template, plus refactored the variables that no longer need to be passed from the controller. It'll be easy enough to revert this if needed in the future.